### DISCUSSION TITLE
Wallet login problem

### DISCUSSION THREAD

**Discussion by Cemalcanyildiz** [IBAX-io/go-ibax#571] [Q&A] [answered]
Hello, when we register to the platform via Twitter and create a wallet, we generate a key called the account seed and we have a wallet ID. The problem is that we do the same with most crypto wallets and the key seed we generate in case we lose our wallet or use it to import it elsewhere
At first I thought it was my seed but it wasn't. To get my private key, I had to open the platform and find the backup partition. Many users had the same problem. Now I can't login to my wallet.
It has been suggested that I can create another one, but my account is linked to that wallet so I cannot create another wallet for my twitter.
 It would be better if we generate our real account seed during the wallet creation process otherwise we will not be able to access our wallet as I cannot do it now.
Big problem for me and for novice people like me

**Comment by sedat2855** [comment]
UP

**Comment by ibaxio** [comment, marked as answer]
Please refer to this discussion:  #62
